frontend/packages/kubevirt-plugin/src/selectors/service/selectors.ts

```diff
@@ -13,7 +13,7 @@ const getServiceSelectors = (service: ServiceKind) =>
   service && service.spec && service.spec.selector ? service.spec.selector : {};
 
 export const getServicesForVmi = (services: ServiceKind[], vmi: VMIKind): ServiceKind[] => {
-  const vmLabels = getVmiTemplateLabels(vmi);
+  const vmLabels = getVMILabels(vmi);
```

Comment:
could we rather use `getLabels` from `console-shared`?

Reply:
Nice :exclamation: 
done